src/drawer/ExNavigationDrawerLayout.js

```diff
@@ -68,6 +69,10 @@ export default class ExNavigationDrawerLayout extends React.Component {
         <ScrollView contentContainerStyle={styles.navigationViewScrollableContentContainer}>
           {this._renderDrawerItems()}
         </ScrollView>
+
+        <View>
+          {this.props.renderFooter()}
```

Comment:
in my case footer was supposed to be fixed at the bottom of the drawer. Besides I think that if you want to add something to the scrollview maybe you should use a proposed DrawerNavigationChild in #251, am I right?